static/bidder-info/metax.yaml

```diff
@@ -16,3 +16,7 @@ capabilities:
       - video
       - native
       - audio
+userSync:
+  redirect:
+    url: https://cm.metaxads.com/pixel?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redir={{.RedirectURL}}
```

Comment:
Verified new syncURL, executes with 302
`/setuid` is called with `metax` present in the set-cookie `uids`